docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+version: "3"
+services:
+  db:
+    image: postgres:12.3
+    volumes:
+      - db_data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: "postgres"
+      POSTGRES_PASSWORD: "postgres"
+    ports:
+      - "5432:5432"
+
+  redis:
+    image: redis:5.0
+    volumes:
+      - redis_data:/redis_data
+    ports:
+      - "6379:6379"
+
+  authgear:
+    build: .
+    environment:
+      DEV_MODE: "true"
+      LOG_LEVEL: "debug"
```

Comment:
docker-compose by default load `.env` so it is unnecessary.

Reply:
It loads .env to substitute variable references in docker-compose.yaml, but for container environment, either `environment` or `env_file` should be used.